hosts/ghaf-log/loki.nix

```diff
@@ -0,0 +1,48 @@
+# SPDX-FileCopyrightText: 2022-2024 TII (SSRC) and the Ghaf contributors
+# SPDX-License-Identifier: Apache-2.0
+_: let
+  loki_data_dir = "/var/lib/loki";
+in {
+  systemd.tmpfiles.rules = [
+    "d /var/lib/loki 0777 loki loki - -"
+  ];
+
+  services.loki = {
+    enable = true;
+    configuration = {
+      auth_enabled = false;
```

Comment:
Hi @joinemm can we add basic authentication with loki service? I think its possible to do with `nginx` as described in article [here](https://dev.to/ruanbekker/running-loki-behind-nginx-reverse-proxy-1699)
This to match credentials what device will be sending from device [here](https://github.com/tiiuae/ghaf/pull/625/files#diff-87d86a1e265f336e25adef2e8f43d3e185ddb94bbf7c68e8e7a59d0befa1c9fcR35-R38)

Reply:
yes but should we merge this first